cup_of_join.py

```diff
@@ -0,0 +1,8 @@
+def join(*parameters, sep='-'):
+    l = []
+    if not parameters:
+        return None
+    for i in range(len(parameters)-1):
```

Comment:
Is there a way to simplify your code here? (what about using list comprehension?)